spaces/domains.json

```
@@ -87,6 +87,7 @@
   "gov.artha.finance": "arthadao.eth",
   "governance.dfyn.network": "dfyn.eth",
   "signal.aave.com": "aave.eth",
+  "vote.singularitydao.ai":"sdao.eth",
```

Please add a space before your ENS value.
```suggestion
  "vote.singularitydao.ai": "sdao.eth",
```